File: backup/Twitch/app/src/main/java/com/exact/twitch/ui/streams/StreamsFragment.java
package com.exact.twitch.ui.streams;

import android.os.Bundle;
import androidx.annotation.NonNull;
import androidx.annotation.Nullable;
import android.view.View;

public class StreamsFragment extends BaseStreamsFragment {

    private String game;

    @Override
    public void onViewCreated(@NonNull View view, @Nullable Bundle savedInstanceState) {
        super.onViewCreated(view, savedInstanceState);
        if (getArguments() != null) { //init in onViewCreated, because if view is not visible it's not needed
            game = getArguments().getString("game");
        }
    }

    @Override
    protected void initData() {
        getViewModel().loadStreams(game, null, "live");
    }
}

File: app/src/main/java/com/exact/xtra/ui/streams/StreamsFragment.kt
package com.exact.xtra.ui.streams

import android.os.Bundle
import android.view.View
import com.exact.xtra.model.game.Game

class StreamsFragment : BaseStreamsFragment() {

    private var game: Game? = null

    override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
        super.onViewCreated(view, savedInstanceState)
        game = arguments?.getParcelable("game")
    }

    override fun loadData(override: Boolean) {
        viewModel.loadStreams(game?.info?.name)
    }
}
